docs/auth.md

```diff
@@ -263,6 +265,60 @@ Content-Type: application/json
 
 ### PUT /auth/register/:client-id
 
+This route is used by the clients to update informations about them-selves.
+The client has to sent its registration access token to be able to use this
```

Comment:
Thanks, it's fixed in https://github.com/cozy/cozy-stack/pull/188/commits/eaae16d61f970f7882c067b93f8b85e8d2089154